frontend/src/static/js/v_authorship.js

```diff
@@ -253,6 +293,10 @@ window.vAuthorship = {
   },
 
   computed: {
+    filterSearchDisplay() {
+      return this.filterSearch === '*' ? '' : this.filterSearch;
```

Comment:
I do not feel we even need this new computed property. Just filterSearch should be enough and the logic can be simplified. The only reason we want the initial value of filterSearch to be * is because we need this for the minimatch library. Thus, it is all right to set filterSearch = '' in data and bind input v-model = filterSearch. Then in the `selectedFiles` we change the filter function to `minimatch(file.path, this.filterSearch || '*', { matchBase: true, dot: true }))` so empty string will match all file names.

Reply:
👍 Changed the 'computed property' to `this.filterSearch || '*'` instead.
Changed `filterSearch` -> `searchBarValue` as well since its not referring to the value used in `minimatch` anymore as well.

> bind input v-model = filterSearch

This works, but the likely reason it was done otherwise (before this pr) is that it would cause `selectedFiles` to change on every key type, which causes performance to tank